tmail-backend/pom.xml

```diff
@@ -236,647 +195,49 @@
-                <groupId>${james.groupId}</groupId>
-                <artifactId>james-server-cassandra-guice</artifactId>
-                <version>${james.version}</version>
-                <type>test-jar</type>
-            </dependency>
-            <dependency>
-                <groupId>${james.groupId}</groupId>
-                <artifactId>james-server-cassandra-ldap-guice</artifactId>
-                <version>${james.version}</version>
-            </dependency>
-            <dependency>
-                <groupId>${james.groupId}</groupId>
-                <artifactId>james-server-cassandra-ldap-guice</artifactId>
-                <version>${james.version}</version>
-                <type>test-jar</type>
-            </dependency>
-            <dependency>
-                <groupId>${james.groupId}</groupId>
-                <artifactId>james-server-cassandra-rabbitmq-guice</artifactId>
-                <version>${james.version}</version>
-            </dependency>
-            <dependency>
-                <groupId>${james.groupId}</groupId>
-                <artifactId>james-server-cassandra-rabbitmq-guice</artifactId>
-                <version>${james.version}</version>
-                <type>test-jar</type>
-            </dependency>
-            <dependency>
-                <groupId>${james.groupId}</groupId>
-                <artifactId>james-server-cli</artifactId>
-                <version>${james.version}</version>
-            </dependency>
-            <dependency>
-                <groupId>${james.groupId}</groupId>
-                <artifactId>james-server-data-api</artifactId>
-                <version>${james.version}</version>
-            </dependency>
-            <dependency>
-                <groupId>${james.groupId}</groupId>
-                <artifactId>james-server-data-ldap</artifactId>
-                <version>${james.version}</version>
-                <type>test-jar</type>
-            </dependency>
-            <dependency>
-                <groupId>${james.groupId}</groupId>
-                <artifactId>james-server-data-memory</artifactId>
-                <version>${james.version}</version>
-            </dependency>
-            <dependency>
-                <groupId>${james.groupId}</groupId>
-                <artifactId>james-server-guice-cassandra</artifactId>
-                <version>${james.version}</version>
-            </dependency>
-            <dependency>
-                <groupId>${james.groupId}</groupId>
-                <artifactId>james-server-guice-configuration</artifactId>
-                <version>${james.version}</version>
-            </dependency>
-            <dependency>
-                <groupId>${james.groupId}</groupId>
-                <artifactId>james-server-guice-common</artifactId>
+                <artifactId>james-project</artifactId>
                 <version>${james.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
```

Comment:
Please maven do take the deoendency management section of the pom.xml of `james-project` and put it here as if I had been copy and pasting it.

Reply:
#72laBalle

Reply:
Why need to import the BOM of james-server-guice then?I think all submodules of james-server-guice are being declared in the james-project maven file no?